.c3i/config_v1.yml

```diff
@@ -42,6 +42,8 @@ tasks:
       - name: "continuous-integration/jenkins/pr-merge"
       - name: "c3i/conan-v2/pr-merge"
         required_for_references: "conan_v2_ready_references.yml"
+  build_single_reference:
+    package_id_cache_type: "artifactory"
```

Comment:
So this one is staying? I thought both would be "none" :)

Reply:
Sorry, this was a leftover from other changes. I have changed it to "none" (although it is the default), so we are sure this is the value